frontend/src/components/profileList/BlockUser.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div v-if="isBlocked(userId) === true">
+       <a href="#" v-on:click.prevent="updateRelationship(userId, ValidRelationships.NONE)" >Unblock</a>
```

Comment:
did you see [$emit()](https://vuejs.org/guide/components/events.html#emitting-and-listening-to-events)?
Not sure if that applies here, as always, I'm in the train and can't build a new container...

Reply:
Yes, I din't understand it completely, if it's an improvement on the current implementation, please let me know so I can dive into it, like I mentioned, there are 15 ways to do things, and if there's one that works and that i understand I run with that until further notice :)

Reply:
I believe it's meant for emitting events to the parent component/view
so if the ListRow component has an "updateRelationship" method, you could emit 
```vue
<a href="#" @click.prevent="$emit('updateRelationship', userId, id_of_new_relation)">
```
and have the parent component handle the updating

But I also realize now that the updateRelationship is a method/action from the userStore, so not sure how that ties together..


just leave it as is I guess :P